### ISSUE TITLE
Departure

### ISSUE THREAD

**Issue by bigznahar** [altmank/CaptainOfCheats#28]
##Note
Crashes at the moment of loading.

**Describe the bug**
Hi! Happy New Year! Listen, help with the mod, please. Crashes at the moment of loading the map, and throws it into the game menu, thank you very much in advance. The patch of the game coincides with your modo, on 0.4.13

**Logs**
Please provide a log file from the directory %USERPROFILE%\Documents\Captain of Industry\Logs

**Comment by altmank** [comment]
Unable to reproduce or troubleshoot based on the provided information and lack of game logs. Exact steps to reproduce, and a log file are both going to increase the chances I can identify an issue.

**Comment by bigznahar** [comment]
[23-01-02_20-50-28_7877 Launching with a mod.log](https://github.com/altmank/CaptainOfCheats/files/10333066/23-01-02_20-50-28_7877.Launching.with.a.mod.log)
[23-01-02_21-14-07_8894 This magazine is run without a mod.log](https://github.com/altmank/CaptainOfCheats/files/10333068/23-01-02_21-14-07_8894.This.magazine.is.run.without.a.mod.log)

**Comment by bigznahar** [comment]
Captain of Industry v0.4.13 Hi, thanks for answering. I launch with your updated mod, for the new version of the game, 1.Step-I launch the game, when the menu appears, I select any map, without even changing the settings, I reach the launch menu tab, there is a check mark of your mod, I launch the map, when loading the map, somewhere in the middle of loading there is a departure to the game menu, with the error log, I will now throw it off. " The step is all the same, I just run it by removing the check mark from the mod, and it loads completely.

**Comment by bigznahar** [comment]
I can't figure out where to get more logs, logs about the problem, except in the documents

**Comment by bigznahar** [comment]
[Player.log](https://github.com/altmank/CaptainOfCheats/files/10333123/Player.log)

**Comment by bigznahar** [comment]
Failed to find location for unlock of Technology Fruit Seeds! Relaxing constraints! 
here's what I read in the log file, maybe it will help you?

**Comment by altmank** [comment]
Hi, unfortunately I am still unable to reproduce the issue, even with the additional steps you provided. 

The log indicates an issue but its not an issue I can reproduce and am not seeing with my installation to allow me to troubleshoot further.

I don't have any other reports of this issue so I can only really offer general advice that may or may not help. 

Please consider using Steam to verify integrity of the COI game files, perhaps there is something in your local installation affecting mod loading. Additionally, please try deleting this mods folder entirely and downloading/reinstalling it.

**Comment by bigznahar** [comment]
Hi, thank you for your answer and help. Even though what you said didn't help me. Deleted the game, deleted all files from the game. I bet everything for a new one. I don't have a Steam version of the game, a regular repack. Unfortunately, I don't have the opportunity to buy a license. If we compare versions 0.4.12 and your mod 1.9.0, everything goes with a bang, but with 1.9.1 and 0.4.13 such troubles. I've watched a video about the new update here and a map has been added there, but I don't have it, maybe that's the point, although the version of the game is 0.4.13 at the bottom. Well, buddy, you still bow for a good mod. As they say, we have Russian health to you and good luck
